spec.html

```diff
@@ -49105,17 +49078,17 @@ <h1>Tear Free Reads</h1>
       </emu-note>
     </emu-clause>
 
-    <emu-clause id="sec-memory-order" aoid="memory-order">
+    <emu-clause id="sec-memory-order">
       <h1>Sequentially Consistent Atomics</h1>
-      <p>For a candidate execution _execution_, memory-order is a strict total order of all events in EventSet(_execution_) that satisfies the following.</p>
+      <p>For a candidate execution _execution_, <dfn>is-memory-order-before</dfn> is a strict total order of all events in EventSet(_execution_) that satisfies the following.</p>
       <ul>
-        <li>For each pair (_E_, _D_) in _execution_.[[HappensBefore]], (_E_, _D_) is in memory-order.</li>
+        <li>For each pair (_E_, _D_) in _execution_'s happens-before, _E_ is-memory-order-before _D_ in _execution_.</li>
```

Comment:
You get the idea at this point:

```suggestion
        <li>For each event _E_ and _D_, _E_ is-memory-order-before _D_ in _execution_ if _E_ happens-before _D_ in _execution_.</li>
```